src/app/models/form.server.model.ts

```diff
@@ -353,6 +353,10 @@ const compileFormModel = (db: Mongoose): IFormModel => {
               'Webhook must be a valid URL over HTTPS and point to a public IP.',
           },
         },
+        isRetryEnabled: {
```

Comment:
Worried this turns into the `hasAllowedEmailDomains` and `allowedEmailDomains` thingamagic. Do you foresee us adding some sort of `retryCount` parameter to the model? If not then this is gtg

Reply:
i think it should be fine because we don't intend the feature to be a form-specific parameter

Reply:
nope, I don't foresee having to add such a thing. I don't see a use case for allowing admins to customise the retry format.